bin/snapshot-backups/README-DR.md

```diff
@@ -12,33 +115,22 @@
         For example:
         USAGOV-784-defacement-recovery.dev.4250.process_test_001
 
-### Static site backup
-        export echo=echo
+### Manually Tagged Static site backup
+
+        echo=echo   ### Do NOT export echo. Ask me how I know this (tm).
+        dryrun='--dryrun'
         $echo bin/snapshot-backups/site-snapshot-create $BACKUP_TAG
         $echo bin/snapshot-backups/site-snapshot-download $BACKUP_TAG
         $echo bin/snapshot-backups/site-snapshot-list
 
-### DB backup
-        export echo=echo
+### Manually Tagged DB backup
+
+        echo=echo   ### Do NOT export echo. Ask me how I know this (tm).
+        dryrun='--dryrun'
         $echo bin/snapshot-backups/db-dump-download $BACKUP_TAG
         $echo bin/snapshot-backups/db-dump-push-to-snapshot $BACKUP_TAG
-        $echo bin/snapshot-backups/db-list
-
-## 2. Snapshot backup using helper (creates tag from Branch, Space, Suffix. Then runs command)
+        $echo bin/snapshot-backups/db-snapshot-list
 
-### See *bin/deploy/includes*, specifically the functions *assertSpace,  spaceCCIContainerTag* and *createSpaceAssertedBackupTag* for details of how the *stw* (snapshoot tool wrapper) script assembles the backup tag, and asserts that the currect space matches the arguments provided to *stw*
-
-### Static site backup
-        export echo=echo
-        $echo bin/snapshot-backups/stw $SPACE $BRANCH $SUFFIX site-snapshot-create
-        $echo bin/snapshot-backups/stw $SPACE $BRANCH $SUFFIX site-snapshot-download
-        $echo bin/snapshot-backups/site-snapshot-list
-
-### DB backup
-        export echo=echo
-        $echo bin/snapshot-backups/stw $BACKUP_TAG db-dump-download
-        $echo bin/snapshot-backups/stw $BACKUP_TAG db-dump-push-to-snapshot
-        $echo bin/snapshot-backups/db-list
 ## 3. Snapshot restore
 
-## 4. Snapshot restore using helper
+TBD
```

Comment:
Is this actually not implemented (site-snapshot-deploy looks promising), or just not documented?

Reply:
Just not documented yet.